src/db/oracledb/identities-dao.js

```diff
@@ -0,0 +1,70 @@
+import _ from 'lodash';
+
+import { serializeIdentities, serializeIdentity } from 'serializers/identities-serializer';
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+
+/**
+ * Return parsed query with keys changed to remove the "filter[]" wrapper
+ *
+ * @param {object} query query parameters
+ * @returns {object} parsed query
+ */
+const parseQuery = (query) => {
```

Comment:
Should this go in `utils` in the skeleton? I imagine other APIs could use this including some persons endpoints I haven't gotten to yet.

Reply:
Good idea, I'll create a PR in skeleton.